src/qc/calculate_qc_metrics/script.py

```diff
@@ -50,7 +60,15 @@ def main():
             if not qc_metric in var:
                 raise ValueError(f"Value for --var_qc_metrics, {qc_metric} "
                                  f"not found in .var for modality {par['modality']}")
-            qc_column = var[qc_metric].values
+            qc_column = var[qc_metric]
+            if qc_column.isna().any():
+                if not par["var_qc_metrics_fill_na_value"]:
```

Comment:
Should this not be

```suggestion
                if par["var_qc_metrics_fill_na_value"] is None:
```

Otherwise passing `--var_qc_metrics_fill_na_value false` will not produce the intended result.

Reply:
Good catch! Forgot to change this from the previous implementation 👍